timescaledb/how-to-guides/connecting/about-psql.md

```diff
@@ -26,11 +26,20 @@ Alternatively, you can use a service URL to provide the details, like this:
 psql postgres://<USERNAME>@<HOSTNAME>:<PORT>/<DATABASENAME>?sslmode=require
 ```
 
+If you configured Timescale Cloud service to connect with stricter [SSL mode] [ssl-mode],
+then use:
+
+```shell
+    psql "postgres://tsdbadmin@$SERVICE_URL_WITH_PORT/tsdb?sslmode=verify-full"
+    
+```
```

Comment:
The spacing is out here, and it should be `bash` not `shell`, I think.

Reply:
done!!!